settings.py

```diff
@@ -6,4 +6,11 @@
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-sequence: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-MONGODB_URL = os.getenv("MONGODB_URL", "mongodb://localhost:27017")
+
+DB_HOST = os.getenv('POSTGRES_HOST', '127.0.0.1')
```

Comment:
As we're moving away from default values, this probably shouldn't have defaults.